docs/user/pings/ping-schedules-and-timings.md

```diff
@@ -0,0 +1,21 @@
+# Ping schedules and timings overview
+
+Full reference details about the [`metrics`](metrics.html) and [`baseline`](baseline.html) ping schedules are detailed elsewhere.
+
+The following diagram shows a typical timeline of a mobile application, when pings are sent and what timing-related information is included.
+
+![ping timeline diagram](ping_timing.svg)
+
+There are two distinct runs of the application, where the OS shutdown the application at the end of Run 1, and the user started it up again at the beginning of Run 2.
+
+There are three distinct foreground sessions, where the application was visible on the screen and the user was able to interact with it.
+
+The rectangles for the baseline and metrics pings represent the "ping lifetimes" of those pings, which always start exactly at the end of the preceding ping.  The `ping_info.start_time` and `ping_info.end_time` metrics included in these pings correspond to these ping lifetimes.
```

Comment:
```suggestion
The rectangles for the `baseline` and `metrics` pings represent the measurement windows of those pings, which always start exactly at the end of the preceding ping.  The `ping_info.start_time` and `ping_info.end_time` metrics included in these pings correspond to these beginning and the end of their measurement windows.
```

We use "measurement window" as the term to describe "ping lifetime" in other docs. Does this make sense?

Reply:
Totally makes sense.  Thanks.